doc/manual/manual.tex

```diff
@@ -8168,7 +8168,53 @@ \subsubsection{Running benchmarks that require code}
+
+\begin{align*}
+T'(x,y) = T_0'\cos\left(\frac{2\pi x}{\lambda}\right)\sin\left(\frac{\pi y}{b} \right)
+\end{align}
+
+Note that because we place the bottom boundary of the domain at $y=0$ and the top at $y=b$, the perturbation vanishes at the top and bottom boundaries. This departs slightly from the setup in \cite{TS14} where the top and bottom boundaries of the domain are at $y=\pm b/2$.
+
+The analytic expression for the critical Rayleigh number, $Ra_{cr}$ is given in Turcotte and Schubert \cite{TS14} equation 6.319:
+
+\begin{align*}
+Ra_{cr}=\frac{\left(\pi^2+\frac{4\pi^2 b^2}{\lambda^2}\right)^3}{\frac{4\pi^2 b^2}{\lambda^2}}
+\end{align}
+
+The linear stability analysis also makes a prediction for the dimensionless growth rate of the instability $\alpha'$ (Turcotte and Schubert \cite{TS14} equation 6.315). The maximum vertical velocity is given by:
+
+\begin{align*}
+v_{y,max} = \frac{2\pi}{\lambda}\phi_0' e^{\alpha' t},
+\end{align}
+
+where
+
+\begin{align*}
+\phi_0' = -\frac{2\pi}{\lambda}\frac{\rho_0 g \alpha T_0'}{\mu}\left(\frac{4\pi^2}{\lambda^2}+\frac{\pi^2}{b^2} \right)^{-2},
+\end{align}
+
+and
+
+\begin{align*}
+\alpha'=\frac{\kappa}{b^2}\left[\frac{\rho_0 g \alpha b^3 \Delta T}{\mu \kappa}\left(\frac{\frac{4\pi^2 b^2}{\lambda^2}}{\left(\frac{4\pi^2 b^2}{\lambda^2}+\pi^2\right)^2}\right) -\left(\pi^2+\frac{4\pi^2b^2}{\lambda^2}\right)\right].
+\end{align}
+
+We use bisection to determine $Ra_{cr}$ for specific choices of the domain geometry, keeping the depth $b$ constant and varying the domain width $\lambda$. If the vertical velocity increases from the first to the second timetsp, the system is unstable to convection. Otherwise, it is stable. Each calculation is terminated after the second timestep. Fig. \ref{fig:onset-1} shows the numerically-determined threshold for the onset of convection, which can be compared directly with the theoretical prediction (green curve) and Fig. 6.39 of \cite{TS14}. The relative error between the numerically-determined value of $Ra_{cr}$ and the analytic solution are shown in the right panel of Fig \ref{fig:onset-1}.
+
+\begin{figure}
+ \includegraphics[width=0.49\textwidth]{cookbooks/benchmarks/onset-of-convection/racr.png}
+ \hfill
+ \includegraphics[width=0.49\textwidth]{cookbooks/benchmarks/onset-of-convection/racr_error.png}
+ \caption{(left) Comparison of numerically-determined and theoretical values for $Ra_{cr}$. Red circles indicate numerical simulations unstable to convection, black circles indicate simulations that are stable. The green dashed curve indicates the theoretical prediction. (right) Relative error in determination of $Ra_{cr}$. Dashed red line indicates error tolerance used in bisection procedure.}
```

Comment:
I think the typical style with other figures in the manual is to say `Left: ... . Right: ...`